pulp_rpm/app/modelresource.py

```diff
@@ -0,0 +1,115 @@
+from pulpcore.app.modelresource import QueryModelResource
+from pulp_rpm.app.models import (
+    DistributionTree,
+    Modulemd,
+    ModulemdDefaults,
+    Package,
+    PackageCategory,
+    PackageGroup,
+    PackageEnvironment,
+    PackageLangpacks,
+    RepoMetadataFile,
+    UpdateRecord,
+)
+
+
+# class <model-name>Resource(QueryModelResource):
+#    def set_up_queryset(self):
+#        return <model>.objects.filter(pk__in=self.repo_version.content)
+#
+#     class Meta:
+#         model = <model>
```

Comment:
Did you intentionally leave this in? Just wanted to confirm because I didn't see in the pulp_file PR.

Reply:
I did, but probably shouldn't have - let me remove it and store it elsewhere (like, say, the docs I need to write...)